python/sglang/srt/two_batch_overlap.py

```diff
@@ -31,7 +32,11 @@ def compute_split_seq_index(
     forward_mode: "ForwardMode",
     num_tokens: int,
     extend_lens: Optional[Sequence[int]],
+    draft_token_num_per_batch: Optional[int],
 ) -> Optional[int]:
+    if forward_mode.is_target_verify():
```

Comment:
nit: (not only for this line but for whole pr) it seems target verify behaves somehow like a generalization of normal decode, thus shall we somehow unify the mtp branch and the decode branch. For example, here we may unify w/ the is_decode branch as:

```
if is_target_verify or is_decode:
  return (num_tokens // token_num_per_batch) // 2
```

where token_num_per_batch is the draft token num, or exactly 1 in decode.